qa-test3.planx-pla.net/portal/gitops.json

```diff
@@ -49,38 +49,35 @@
     "index": {
       "introduction": {
         "heading": "NIAID Data Ecosystem Datasets",
-        "text": "Disciplines within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem. The data within projects are findable, accessible, interoperable, and reusable (FAIR), but it will be easier for each individual research community to work within the system.", 
+        "text": "Disciplines within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem, facilitating individual research communities to work within the system. The data within all supported resources are findable, accessible, interoperable, and reusable (FAIR).", 
         "link": "/datasets"
       },
       "buttons": [
         {
           "name": "ImmPort",
-          "body": "ImmPort is funded by the NIH, NIAID, and DAIT in support of the NIH mission to share data with the public. Data is provided by NIH-funded programs and other research organizations.",
+          "body": "Explore ImmPort, a resouce funded by the NIH, NIAID, and DAIT in support of the NIH mission to share data with the public. Data is provided by NIH-funded programs and other research organizations.",
           "external_link": "https://www.immport.org/home",
           "logo": "/custom/logo/immport-main-icon.png"
         },
         {
           "name": "NIAID TB Environment",
           "icon": "data-explore",
-          "body": "The NDE TB Environment.",
+          "body": "Explore data from the NIAID TB Portals Program and PATRIC, the all-bacterial Bioinformatics Database and Analysis Resource Center.",
```

Comment:
do you maybe mean "OCICB and PATRIC"?

Reply:
So OCICB is the Office of Cyber Infrastructure and Comp Bio at NIAID. The data that we are actually hosting is from TB Portals which I believe the OCICB administrates.